elkscmd/cron/crontab.c

```diff
@@ -219,15 +219,15 @@ visual(struct passwd *pwd)
             exit(0);
         seteuid(getuid());
         do {
-            fprintf(stderr, "Do you want to retry the same edit? ");
+            fputs("Do you want to retry the same edit? ", stderr);
```

Comment:
Same here as commented above.

Reply:
Turns out that this doesn't matter (function and binary are same size either way) since something else is already pulling fputs in.  In general, fputs should take fewer cycles when no formatting is required.  But since I don't know how/why fputs is getting pulled in already (maybe it can be trimmed out...) I'll revert this for now.